### ISSUE TITLE
step with full action url being rewritten to DEFAULT_ACTIONS_URL (from https://gitea.com to https://github.com)

### ISSUE THREAD

**Issue by cakekindel** [go-gitea/gitea#25615]
### Description

```yaml
    steps:
      - uses: https://gitea.com/actions/checkout@v2
      - uses: https://gitea.com/actions/cache@v3
```

in the runner this is rewritten to a gitea root URL:

```txt


Jun 30 22:18:20 srv act_runner[1096137]: [release/release]   ☁  git clone 'https://github.com/actions/cache' # ref=v3
--


Jun
 30 22:18:34 srv act_runner[1096137]: [release/release] [DEBUG] Cloned 
https://github.com/actions/cache to 
/home/gitea_runner/.cache/act/https---gitea.com-actions-cache@v3
```

### Gitea Version

1.20.0+rc0, act_runner version v0.2.0-10-gcf48ed8

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

https://gist.orionkindel.com/orion/be4f0da9bab84e2fa94f8f0dedbabe81

### Operating System

debian

### How are you running Gitea?

docker compose, act_runner is a daemon with rootless docker context

### Database

SQLite

**Comment by puni9869** [comment]
Assign me.

**Comment by satiAC** [comment]
fixed, waiting for merge
https://gitea.com/gitea/act/pulls/71

**Comment by lunny** [comment]
Closed as https://gitea.com/gitea/act/pulls/71 merged.

**Comment by wolfogre** [comment]
Thanks for reporting and fixing. It should be fixed after v0.2.3 and nightly version of act runner.
